Remove graduated feature gate `ShootForceDeletion` (#12078)
gardener/gardener: docs/usage/shoot-operations/shoot_operations.md

## Force Deletion

> When a Shoot fails to be deleted normally, users can force-delete the Shoot by meeting the following conditions:

Reviewer: ```suggestion
When a Shoot fails to be deleted normally, users can force-delete the Shoot if it meets the following conditions:
```

> When a Shoot fails to be deleted normally, users can force-delete the Shoot by meeting the following conditions:

Reviewer 2: ```suggestion
When a Shoot fails to be deleted normally, users can force-delete the Shoot if the following conditions are met:
```
